docs/release_notes/template.md

```diff
@@ -0,0 +1,178 @@
+  
+# Dapr $dapr_version
+
+We're happy to announce the release of Dapr $dapr_version!
+
+We would like to extend our thanks to all the new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+Docs have been updated with all the new features and changes of this release. To get started with new capabilities introduced in this release, go to the [Concepts](https://github.com/dapr/docs/tree/master/concepts) and the [How To section](https://github.com/dapr/docs/tree/master/howto#how-tos).
+
+$warnings
+
+See [this](#upgrading-to-dapr-$dapr_version) section on upgrading Dapr to version $dapr_version.
+
+## New in this release
+
+$dapr_changes
+
+## Upgrading to Dapr $dapr_version
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo:
```

Comment:
```suggestion
Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if docker command needs sudo:
```